src/EnergyPlus/SingleDuct.cc

```diff
@@ -1791,8 +1791,11 @@ namespace SingleDuct {
 		//Do a check and make sure that the max and min available(control) flow is
 		//  between the physical max and min while operating.
 		SysInlet( SysNum ).AirMassFlowRateMaxAvail = min( Sys( SysNum ).AirMassFlowRateMax, Node( InletNode ).MassFlowRateMaxAvail );
-		SysInlet( SysNum ).AirMassFlowRateMinAvail = min( max( Node( OutletNode ).MassFlowRateMin, Node( InletNode ).MassFlowRateMinAvail ), SysInlet( SysNum ).AirMassFlowRateMaxAvail );
-
+		if ( Sys( SysNum ).SysType_Num == SingleDuctConstVolReheat ) { // AirTerminal:SingleDuct:ConstantVolume:Reheat
+			SysInlet( SysNum ).AirMassFlowRateMinAvail = min( max( Node( OutletNode ).MassFlowRateMin, Node( InletNode ).MassFlowRateMinAvail ), SysInlet( SysNum ).AirMassFlowRateMaxAvail );
+		} else { // #6173
+			SysInlet( SysNum ).AirMassFlowRateMinAvail = min( Node( OutletNode ).MassFlowRateMin, SysInlet( SysNum ).AirMassFlowRateMaxAvail );
+		}
```

Comment:
@rraustad The use of inlet node MinAvail will cause damper wide open. The revised code will force damper position at min as desired. Here are plots by running a test file of VAVplusZoneOAUnit.

Damper position from original run

![old damper](https://user-images.githubusercontent.com/5970656/29369848-c3945fb6-8271-11e7-9fd3-f45f27d747f1.jpg)

Damper position from fix run

![new damper](https://user-images.githubusercontent.com/5970656/29369879-d702379e-8271-11e7-9a39-aca330d44666.jpg)

Zone air temperature from original run

![old zonet](https://user-images.githubusercontent.com/5970656/29369884-e1e13728-8271-11e7-80f1-99991f70d3d5.jpg)

Zone air temperature from fix run
![new zonet](https://user-images.githubusercontent.com/5970656/29369922-f78b8a4c-8271-11e7-8f27-0d9c45c816d4.jpg)